lib/Dialect/HW/HWAttributes.cpp

```diff
@@ -190,20 +190,6 @@ void InnerRefAttr::print(AsmPrinter &p) const {
   p << ">";
 }
 
-/// Get an InnerRefAttr, and add the sym to the op if not already
```

Comment:
Awesome, was looking at this the other day.  One less way to create things, thanks! :+1: